README.md

```diff
@@ -1,83 +1,101 @@
+
+Create a new menu and view it:
+```
+require './lib/menu'
+menu = Menu.new
+menu.view_menu
+```
+
+User story 2:
+
+Add dishes to order:
+```
+require './lib/order.rb'
+order = Order.new
+order.add_dish("Chips", 3)
+order.add_dish("Cheese burger", 2)
+order.display_basket
+```
+
+User story 3:
 
-* Advanced! (have a go if you're feeling adventurous):
-  * Implement the ability to place orders via text message.
+```
+require './lib/order.rb'
+order = Order.new
+order.add_dish("Chips", 3)
+order.add_dish("Cheese burger", 2)
+order.total_cost
+```
 
-* A free account on Twilio will only allow you to send texts to "verified" numbers. Use your mobile phone number, don't worry about the customer's mobile phone.
 
-> :warning: **WARNING:** think twice before you push your **mobile number** or **Twilio API Key** to a public space like GitHub :eyes:
->
-> :key: Now is a great time to think about security and how you can keep your private information secret. You might want to explore environment variables.
+Test coverage
+-----
 
-* Finally submit a pull request before Monday at 9am with your solution or partial solution.  However much or little amount of code you wrote please please please submit a pull request before Monday at 9am
+100%
 
 
-In code review we'll be hoping to see:
+Reference
+-----
+I referred to another student's challenge - https://github.com/RPGrimes/takeaway-challenge
 
-* All tests passing
-* High [Test coverage](https://github.com/makersacademy/course/blob/main/pills/test_coverage.md) (>95% is good)
-* The code is elegant: every class has a clear responsibility, methods are short etc.
 
-Reviewers will potentially be using this [code review rubric](docs/review.md).  Referring to this rubric in advance will make the challenge somewhat easier.  You should be the judge of how much challenge you want this at this moment.
+Time spent on challenge
+-----
+Sunday 9am to 8:30pm
+
 
-Notes on Test Coverage
-------------------
+Reflections and mistakes along the way
+-----
 
-You can see your [test coverage](https://github.com/makersacademy/course/blob/main/pills/test_coverage.md) when you run your tests.
+- I spent a lot of time creating a Dish class and using that to try and populate a menu - this wasn't necessary for the challenge. I could have gone straight to creating a Menu in the Menu class since the Menu didn't need to change over time for the challenge. Better planning up front would have helped with this.
+- I don't have a good understanding on accessing variables and using them across different classes. I need more practice! For example, creating a Menu in the Menu class, I needed to refer to another student's code to work out how I could access the Menu in the order class (in the end it was creating a new instance of Menu and making it an attribute of Order)
+- Understand what tests to write before you know what code you need to meet the requirements of the challenge is hard. Again I need more and more practice of this!
```

Comment:
This is extremely good. It shows you are learning from what you did during this challenge and explains what you'll do next time!